### PR TITLE
Soc 301 integration tests for CRUD functionality

### PR DESCRIPTION
**Pull request by rafalkalinski**
Integration tests for CRUD functionality
 - post and get notification
 - post and delete notification
 - post and update notification

### CODE REVIEW

File: service/notifications/src/test/java/com/wikia/notificationsservice/integration/ApplicationsIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+  @Test
+  public void notificationCreateAndRead() {
+      Client client = ClientBuilder.newClient();
+
+      Notification notification =
+          new Notification("new test note", "testSourceType", "testSourceId", (long) 1);
+
+      Response response = client.target(String
+          .format("http://localhost:%d/inbox/%d", RULE.getLocalPort(), notification.getUserId()))
```
Comment (kvas-damian) [inline service/notifications/src/test/java/com/wikia/notificationsservice/integration/ApplicationsIntegrationTest.java]: please extract URL to constant

File: service/notifications/src/test/java/com/wikia/notificationsservice/integration/ApplicationsIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
+package com.wikia.notificationsservice.integration;
+
+import com.squarespace.jersey2.guice.BootstrapUtils;
```
Comment (sqreek) [inline service/notifications/src/test/java/com/wikia/notificationsservice/integration/ApplicationsIntegrationTest.java]: Please update your [code style file]
(https://github.com/Wikia/guidelines/tree/master/Java)

File: service/notifications/src/test/java/com/wikia/notificationsservice/integration/ApplicationsIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+//    @Test
+//  uncomment above when functionality is ready
+    public void notificationCreateAndDelete() {
+        Client client = ClientBuilder.newClient();
+
+        Notification notification =
+            new Notification("new test note", "testSourceType", "testSourceId", (long) 1);
+
+        Response response = client.target(String
+            .format("http://localhost:%d/inbox/%d", RULE.getLocalPort(), notification.getUserId()))
```
Comment (sqreek) [inline service/notifications/src/test/java/com/wikia/notificationsservice/integration/ApplicationsIntegrationTest.java]: http://localhost:%d/ can be extracted to const

File: service/notifications/src/test/java/com/wikia/notificationsservice/integration/ApplicationsIntegrationTest.java [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+        response = client.target(String.format("http://localhost:%d%s", RULE.getLocalPort(), url))
+            .request()
+            .get();
+
+        assertThat(response.getStatus()).isEqualTo(Response.Status.NOT_FOUND.getStatusCode());
+    }
+
+    //  @Test
+    //  uncomment above when functionality is ready
```
Comment (sqreek) [inline service/notifications/src/test/java/com/wikia/notificationsservice/integration/ApplicationsIntegrationTest.java]: This should go in or go out.
No dead code please :wink:

File: service/notifications/src/test/java/com/wikia/notificationsservice/integration/ApplicationsIntegrationTest.java (hunk 2 above)
Comment (kvas-damian) [inline service/notifications/src/test/java/com/wikia/notificationsservice/integration/ApplicationsIntegrationTest.java]: Imports are already ordered according to current code styles.

File: service/notifications/src/test/java/com/wikia/notificationsservice/integration/ApplicationsIntegrationTest.java (hunk 4 above)
Comment (rafalkalinski) [inline service/notifications/src/test/java/com/wikia/notificationsservice/integration/ApplicationsIntegrationTest.java]: Functionality is under development. I will take and put these two method in some separate branch.

**Comment by sqreek** [comment]
All GTM

**Pull request merged**